server/src/main/java/org/opensearch/action/support/broadcast/BroadcastRequest.java

```diff
@@ -52,6 +52,8 @@ public class BroadcastRequest<Request extends BroadcastRequest<Request>> extends
     protected String[] indices;
     private IndicesOptions indicesOptions = IndicesOptions.strictExpandOpenAndForbidClosed();
 
+    protected boolean shouldCancelOnTimeout = false;
+
     public BroadcastRequest(StreamInput in) throws IOException {
```

Comment:
wouldn't you need changed for serialization/ deserialization to handle shouldCancelOnTimeout flag based on version